domains/j.json

```
@@ -7,6 +7,15 @@
     "discord": "1117482901353812088"
   },
   "record": {
-    "CNAME": "juststudio7.github.io"
+    "A": [
+      "185.199.108.153",
+      "185.199.109.153",
+      "185.199.110.153",
+      "185.199.111.153"
+    ],
+    "MX": ["mx1.improvmx.com.", "mx2.improvmx.com."],
```

Remove the dots since our config already adds it for you
```suggestion
    "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
```